ValeStyles/Couchbase/An.yml

```diff
@@ -4,8 +4,7 @@ link: https://docs.couchbase.com/styleguide/articles.html
 level: error 
 
 tokens:
-  - '[Aa] \b[aAeEiIoOuU].+\b'
-  - '[Aa] \b[Hh][Tt][Mm][Ll]\b'
```

Comment:
is removing the "A HTML" rule intentional?

Reply:
Ah shit that was a mistake. One sec...